### DISCUSSION TITLE
如图，希望大佬能够好心帮忙解答，万分感谢

### DISCUSSION THREAD

**Discussion by sinLuoT** [mikumifa/biliTickerBuy#164] [提问专区] [answered]
![微信图片_20240627011140](https://github.com/mikumifa/biliTickerBuy/assets/173944417/b04fb332-03af-4ee0-bb71-e35e81b0bd3f)

**Comment by WittF** [comment]
什么问题？

**Reply by sinLuoT** [reply]
接下来怎么办

**Reply by WittF** [reply]
> 接下来怎么办

```
点击下面的网址运行程序
Running on local URL: http://127.0.0.1:7860
Running on public URL: https://1d85b7e72d621453fd.gradio.live

**Reply by sinLuoT** [reply]
左键右键点了之后都没反应

**Reply by WittF** [reply, marked as answer]
> 左键右键点了之后都没反应

按crtl再点试试，或者你可以直接在浏览器访问 http://127.0.0.1:7860

**Reply by sinLuoT** [reply]
![Uploading 微信截图_20240627012843.png…]()

**Reply by sinLuoT** [reply]
edge：嗯… 无法访问此页面127.0.0.1 已拒绝连接。
请尝试：

在 web 上搜索 [127 0 0 1](https://www.bing.com/search?form=ANLKDR&q=127%200%200%201)
检查连接
[检查代理和防火墙](chrome-error://chromewebdata/#buttons)

**Reply by WittF** [reply]
> edge：嗯… 无法访问此页面127.0.0.1 已拒绝连接。 请尝试：
> 
> 在 web 上搜索 [127 0 0 1](https://www.bing.com/search?form=ANLKDR&q=127%200%200%201) 检查连接 检查代理和防火墙

程序运行了吗

**Reply by sinLuoT** [reply]
运行了

**Reply by sinLuoT** [reply]
换成chrome解决了，感谢
